src/storage/activities.tsx

```diff
@@ -25,6 +25,34 @@ export const activityDayGt = (a: ActivitiesDay, b: ActivitiesDay): boolean => {
   return a.date > b.date;
 }
 
+/**
+ * Compares two activity days and returns `true` if they are equal.
+ *
+ * @param a - The first activity day
+ * @param b - The second activity day
+ * @returns a === b
+ */
+const activityDayEq = (a: ActivitiesDay, b: ActivitiesDay): boolean => {
+  return a.date === b.date
+      && a.entries === b.entries
```

Comment:
This would probably require activityEntryEq to be above.
```suggestion
      && a.entries.length === b.entries.length
      && a.entries.every((e, i) => activityEntryEq(e, b.entries[i]))
```

Reply:
Great suggestion! Applied it in the coming commits.